fudge-ratio/src/main/webapp/code/chooseActivities.js

```diff
@@ -0,0 +1,62 @@
+const dropdownText = `
+<div id="dropdown">
+  <p>Select a task and click the start button when you're ready!</p>
+  <label>Choose a task:</label>
+  <select id="options">
+      <option value="" selected>Select one</option>
+      <optgroup label="Schoolwork" id="Schoolwork"></optgroup>
+      <optgroup label="Job-related tasks" id="Job-related tasks"></optgroup>
+      <optgroup label="Housework" id="Housework"></optgroup>
+  </select>
+  <button disabled id="submit">Add activity</button>
+</div>
+  `;
+
+import { get, parent, createFromText } from "./utils.js";
+import { KEYS } from "./save.js";
+
+const getActivities = async () => {
+  const response = await fetch("/data");
```

Comment:
Does that still work (since we do not have a /data anymore) ?

Reply:
I believe it should still work since the Activity servlet still exists with that endpoint.